x-pack/test/functional_basic/apps/ml/permissions/read_ml_access.ts

```diff
@@ -57,6 +58,10 @@ export default function ({ getService }: FtrProviderContext) {
           await ml.testExecution.logTestStep('should load the ML app');
           await ml.navigation.navigateToMl();
 
+          await ml.testExecution.logTestStep('should redirect to the "Data Visualizer" page');
+          const browserURl = await browser.getCurrentUrl();
+          expect(browserURl).to.contain('/ml/datavisualizer');
+
```

Comment:
I think we should have the same test in the [poweruser suite](https://github.com/elastic/kibana/blob/main/x-pack/test/functional_basic/apps/ml/permissions/full_ml_access.ts) as well.

Reply:
Added in https://github.com/elastic/kibana/pull/139469/commits/e25652a16c91ad64e72de1e931ddb72cfb0c1d6b